components/nodemanager-service/pgdb/node-managers.go

```diff
@@ -303,7 +303,7 @@ func (db *DB) GetNodeManagers(sortField string, insortOrder manager.Query_OrderT
 	if err != nil {
 		return nil, 0, errors.Wrapf(err, "GetNodeManagers error validating node manager filters")
 	}
-	whereFilter, err := buildWhereFilter(filters, "nm", mgrFilterField)
+	whereFilter, _, err := buildWhereHavingFilter(filters, "nm", mgrFilterField)
```

Comment:
It seems the second argument here is only ever "nm" or "n", might it be worth it to use some well-named package-level constants:

```golang
const (
    NodeTableAbbrev        = "n"
    NodeManagerTableAbbrev = "nm"
)
```

that is one of a number of small things we could probably do to make the code a bit easier to understand.

Reply:
Yes, I created these constants for the table abbreviations:
```go
const (
  NodesTableAbbrev        = "n"
  TagsTableAbbrev         = "t"
  NodeManagersTableAbbrev = "nm"
)
```